Shaders/PerfectPerspective.fx

```diff
@@ -19,7 +19,7 @@ uniform int FOV <
 	ui_tooltip = "This setting should match \n
 		"your in-game Field of View";
 	ui_type = "drag";
-	ui_min = 45; ui_max = 120;
+	ui_min = 45; ui_max = 120; ui_step = 0.2;
```

Comment:
ui_step is cast to int type by reshade

Reply:
Steps are still min 1, but it takes more mouse movement to change the value, than let's say ui_step=1

Reply:
Steps are still min 1, but it takes more mouse movement to change the value, than let's say ui_step=1